core/math/geometry.h

```diff
@@ -888,6 +888,14 @@ class Geometry {
 		return Vector<Vector<Vector2> >();
 	}
 
+	static real_t get_polygon_area(const Vector<Vector2> &p_polygon) {
+		if (p_polygon.size() < 3)
+			return 0.0;
+		// positive = counterclockwise (boundary)
+		// negative = clockwise (hole)
+		return Triangulate::get_area(p_polygon);
+	}
+
 	static bool is_polygon_clockwise(const Vector<Vector2> &p_polygon) {
```

Comment:
An alternative `is_polygon_clockwise` implementation would be:
```cpp
	static bool is_polygon_clockwise(const Vector<Vector2> &p_polygon) {
		if (p_polygon.size() < 3)
			return false;
		return Triangulate::get_area(p_polygon) < 0.0;
	}
```
But I haven't dared to touch that. The existing implementation bypasses computing an actual area so I think the existing one is better performance-wise (not tested).